[2025-07-21T16:52:44.616+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: complex_failure_dag.start_task manual__2025-07-21T16:52:43.980676+00:00 [queued]>
[2025-07-21T16:52:44.625+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: complex_failure_dag.start_task manual__2025-07-21T16:52:43.980676+00:00 [queued]>
[2025-07-21T16:52:44.625+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2025-07-21T16:52:44.637+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): start_task> on 2025-07-21 16:52:43.980676+00:00
[2025-07-21T16:52:44.644+0000] {standard_task_runner.py:57} INFO - Started process 182 to run task
[2025-07-21T16:52:44.647+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'complex_failure_dag', 'start_task', 'manual__2025-07-21T16:52:43.980676+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/complex_failure_dag.py', '--cfg-path', '/tmp/tmph8_7lkax']
[2025-07-21T16:52:44.649+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask start_task
[2025-07-21T16:52:44.702+0000] {task_command.py:416} INFO - Running <TaskInstance: complex_failure_dag.start_task manual__2025-07-21T16:52:43.980676+00:00 [running]> on host 2cf9a9b2cf2c
[2025-07-21T16:52:44.763+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='metadata_test' AIRFLOW_CTX_DAG_ID='complex_failure_dag' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2025-07-21T16:52:43.980676+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-21T16:52:43.980676+00:00'
[2025-07-21T16:52:44.764+0000] {task_logging_plugin.py:29} INFO - [Task Metadata] (running): {'event': 'TASK_RUNNING', 'timestamp': '2025-07-21T16:52:44.764156', 'dag_id': 'complex_failure_dag', 'dag_run_id': 'manual__2025-07-21T16:52:43.980676+00:00', 'task_id': 'start_task', 'execution_date': '2025-07-21 16:52:43.980676+00:00', 'try_number': 1, 'start_date': '2025-07-21 16:52:44.616586+00:00', 'end_date': 'None', 'state': 'running', 'queue': 'default', 'operator': 'PythonOperator', 'hostname': '2cf9a9b2cf2c'}
[2025-07-21T16:52:44.764+0000] {task_logging_plugin.py:33} INFO - [Task Metadata] (running): {'event': 'TASK_RUNNING', 'timestamp': '2025-07-21T16:52:44.764156', 'dag_id': 'complex_failure_dag', 'dag_run_id': 'manual__2025-07-21T16:52:43.980676+00:00', 'task_id': 'start_task', 'execution_date': '2025-07-21 16:52:43.980676+00:00', 'try_number': 1, 'start_date': '2025-07-21 16:52:44.616586+00:00', 'end_date': 'None', 'state': 'running', 'queue': 'default', 'operator': 'PythonOperator', 'hostname': '2cf9a9b2cf2c'}
[2025-07-21T16:52:44.764+0000] {logging_mixin.py:154} INFO - Starting complex failure DAG
[2025-07-21T16:52:44.764+0000] {logging_mixin.py:154} INFO - Current time: 2025-07-21 16:52:44.764587
[2025-07-21T16:53:04.769+0000] {logging_mixin.py:154} INFO - Start task completed successfully
[2025-07-21T16:53:04.770+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-07-21T16:53:04.786+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=complex_failure_dag, task_id=start_task, execution_date=20250721T165243, start_date=20250721T165244, end_date=20250721T165304
[2025-07-21T16:53:04.798+0000] {task_logging_plugin.py:29} INFO - [Task Metadata] (success): {'event': 'TASK_SUCCESS', 'timestamp': '2025-07-21T16:53:04.798746', 'dag_id': 'complex_failure_dag', 'dag_run_id': 'manual__2025-07-21T16:52:43.980676+00:00', 'task_id': 'start_task', 'execution_date': '2025-07-21 16:52:43.980676+00:00', 'try_number': 2, 'start_date': '2025-07-21 16:52:44.616586+00:00', 'end_date': '2025-07-21 16:53:04.786182+00:00', 'state': <TaskInstanceState.SUCCESS: 'success'>, 'queue': 'default', 'operator': 'PythonOperator', 'hostname': '2cf9a9b2cf2c'}
[2025-07-21T16:53:04.799+0000] {task_logging_plugin.py:33} INFO - [Task Metadata] (success): {'event': 'TASK_SUCCESS', 'timestamp': '2025-07-21T16:53:04.798746', 'dag_id': 'complex_failure_dag', 'dag_run_id': 'manual__2025-07-21T16:52:43.980676+00:00', 'task_id': 'start_task', 'execution_date': '2025-07-21 16:52:43.980676+00:00', 'try_number': 2, 'start_date': '2025-07-21 16:52:44.616586+00:00', 'end_date': '2025-07-21 16:53:04.786182+00:00', 'state': <TaskInstanceState.SUCCESS: 'success'>, 'queue': 'default', 'operator': 'PythonOperator', 'hostname': '2cf9a9b2cf2c'}
[2025-07-21T16:53:04.809+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-07-21T16:53:04.840+0000] {taskinstance.py:2778} INFO - 2 downstream tasks scheduled from follow-on schedule check
